addons/sms/models/sms_sms.py

```diff
@@ -96,7 +100,7 @@ def action_set_error(self, failure_type):
     def action_set_outgoing(self):
         self._update_sms_state_and_trackers('outgoing', failure_type=False)
 
-    def send(self, unlink_failed=False, unlink_sent=True, raise_exception=False):
+    def send(self, unlink_failed=False, unlink_sent=True, raise_exception=False, is_marketing=False):
```

Comment:
At this point everything should be stored on sms model, parameter to remove probably